utils/gen_config.py

```diff
@@ -0,0 +1,174 @@
+# coding=utf-8
+"""Update scan facts in config file given scan data.
+
+Run this script as a command line tool to use csv with data about hosts and
+profiles and report file from rho scan to generate camayoc config file.
+
+.. note::
+
+    This tool reads in your camayoc config file in $XDG_CONFIG_HOME and outputs
+    it to the $CWD with new data. It discards old hosts data but uses auth,
+    profile, and any other data provided in your $XDG_CONFIG_HOME config file
+    (if it exists). This does not modify your $XDG_CONFIG_HOME config file
+    directly, that is left to you. Always backup your old config file before
+    you replace it with the generated one!
+
+Example:
+    .. code-block:: bash
+
+        python gen_config.py --profilecsv profiles.csv --reportcsv report.csv
+"""
+from camayoc import config
+import uuid
+from camayoc.exceptions import ConfigFileNotFoundError
+import os
+import csv
+import yaml
+import argparse
+
+
+base_config = """
+rho:
+  auths:
+    - username: root
+      name: default
+      sshkeyfile: $HOME/.ssh/id_rsa
+"""
+
+
+def get_config():
+    """Gather existing config file or create one.
+
+    If a config file is found by camayoc, use that.
+    Otherwise, create the file and directory if necessary.
+    """
+    try:
+        cfg = config.get_config()
+    except ConfigFileNotFoundError:
+        cfg = yaml.load(base_config)
+
+    return cfg, os.path.join(os.getcwd(), 'config.yaml')
+
+
+def gen_hosts(profilefile, reportfile):
+    """Generate list of hosts from profile and report data.
+
+    Args:
```

Comment:
You can use sphinx fields format here:

```rst
:param profilefile: path to the profile file. The path should be a CSV file with the coloumns:
    * name
    * ip
    * profile
    * auth
:param reportfile: ...
```

Reply:
move the param definition to the end of the docstring (like move the ..note below before the param definitions)